hpx/parallel/executors/auto_chunk_size.hpp

```diff
@@ -55,6 +55,12 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v3)
           : min_time_(rel_time.value().count())
         {}
 
+        template <typename Executor>
+        static bool variable_chunk_size(Executor&)
+        {
+            return false;
+        }
```

Comment:
Why is this necessary? The `executor_parameters_traits` should supply a default implementation of this function, if needed.